src/router/index.js

```diff
@@ -0,0 +1,128 @@
+import {getRoutableSignals, flattenConfig} from './utils'
+import urlMapper from 'url-mapper'
+
+let addressbar
+try {
+  addressbar = require('addressbar')
+} catch (e) {
+  addressbar = {
+    pathname: '/',
+    value: '',
+    origin: '',
+    on() {},
+    removeListener() {}
+  }
+}
+
+export default function Router(options = {}) {
```

Comment:
@schotime already decided to publish router as separate package, but most likely it would live in main repo.